Formula/c/ca-certificates.rb

```diff
@@ -121,7 +121,7 @@ def macos_post_install
   end
 
   def linux_post_install
-    rm_f pkgetc/"cert.pem"
+    rm(pkgetc/"cert.pem")
```

Comment:
We should check that the file exists here, because it won't the first time this formula is installed.

Reply:
Glad you're here! 😅

Reply:
How much "more careful" should we be? For all of the ones in the `post_install` blocks should I do a "if this file exists?" check?

Reply:
@issyl0 I think anyone where we don't know specifically that it should be here: it's fine as-is.